releases/release-7.1.0.md

```diff
@@ -169,6 +222,14 @@ TiDB 版本：7.1.0 (upcoming)
 
     更多信息，请参考[用户文档](/ddl-introduction.md#ddl-相关的命令介绍)。
 
+* 支持无需手动取消 DDL 的平滑升级集群功能 [#39751](https://github.com/pingcap/tidb/issues/39751) @[zimulala](https://github.com/zimulala)**tw:ran-huang**
+
+    在 TiDB v7.1.0 之前的版本中，升级集群时需要先手动取消正在运行或排队的 DDL 任务，并在升级完成后再手动添加这些任务。
+
+    为了提供更平滑的升级体验，TiDB v7.1.0 引入了自动暂停和恢复 DDL 任务的功能。从 v7.1.0 开始，你在升级集群前无需手动取消 DDL 任务。系统会自动暂停正在执行或排队的用户 DDL 任务，等待整个集群完成滚动升级后再自动恢复这些任务，让你可以更加轻松地升级 TiDB 集群。
+
+    更多信息，请参考[用户文档](/smooth-upgrade-tidb.md)。
+
```

Comment:
```suggestion

* 重构数据库审计功能（企业版）

    在 v7.1.0 中，TiDB 企业版重构了数据库审计功能，提供了更细粒度的事件过滤和便捷的过滤条件设置，新增了 JSON 格式的输出文件以及存储文件的控制和管理。

    该功能为企业版特性，要获取数据库审计功能及其文档，请在 [TiDB 产品页面](https://pingcap.com/zh/product/#SelectProduct)**企业版**下点击**立即咨询**与 PingCAP 取得联系。

```
@songrijie @bb7133 @lcwangchao PTAL

Reply:
```suggestion
       在 v7.1.0 中， TiDB 企业版对数据库审计能力做了重构，大幅提升了能力范围，改善了使用体验，以满足企业对数据库安全合规的需要：

    - 引入"过滤器"与"规则"的概念，定义了更细分的审计事件定义，支持更细粒度的审计设置。
    - 支持 JSON 格式的规则定义，设置方式比从前更加友好。
    - 自动日志轮转和空间管理。支持保存时间和日志大小两个维度的设置。
    - 审计日志可以用 TEXT 和 JSON 两种格式写入， 方便与三方工具结合。
    - 支持对日志内容进行脱敏，替换所有字面值，提升安全性。

     数据库审计是 TiDB 企业版的重要功能，为企业提供了一种强大的监管和审计工具，以保证企业的数据安全和合规性。TiDB 企业版的数据库审计功能可以帮助企业管理人员追踪数据库操作的来源和影响，确保数据不被非法窃取或篡改。同时，数据库审计还可以帮助企业遵守各种法规和合规要求，确保企业在法律和道德方面的合规性。对企业信息安全具有非常重要的应用价值。

     该功能为企业版特性，要获取数据库审计功能及其文档，请在 [TiDB 产品页面](https://pingcap.com/zh/product/#SelectProduct)**企业版**下点击**立即咨询**与 PingCAP 取得联系。
```

Reply:
Add in https://github.com/pingcap/docs-cn/pull/13896/commits/cced237419aff328ef28d311b208e00158ba2cd2